src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1250,6 +1250,7 @@ internal static partial class EncodingTable
             28599, // iso-8859-9
             28603, // iso-8859-13
             28605, // iso-8859-15
+            29001, // x-europa
```

Comment:
I assume you are talking about the tool itself. my thinking is to convert this tool to some script (perl or powershell) instead of C# so we don't have to rebuild the tool for different platform. 
for your changes here, you just need to generate the new EncodingTable.Data.cs using the tool and you can modify the test accordingly.